packages/firestore/test/unit/local/indexeddb_persistence.test.ts

```diff
@@ -262,6 +264,7 @@ describe('IndexedDbSchema: createOrUpgradeDb', () => {
 
     return withDb(2, db => {
       return db.runTransaction(
+        TEST_ACTION,
```

Comment:
Is it possible to use the current test name, description, or similar here? This would benefit test failures similarly to what you're proposing for end users.

Reply:
We have to convert the tests to use proper functions, since they will be associated with Mocha's `this` context, which then provides the test name. I updated this file and SimpleDbTest.